.eslintrc.js

```diff
@@ -25,6 +25,7 @@ module.exports = {
     'import/newline-after-import': 2,
     'import/no-duplicates': 2,
     'import/no-extraneous-dependencies': 2,
+    '@typescript-eslint/no-extra-semi': 0,
```

Comment:
Please do not disable this rule.

Reply:
```  useEffect(() => {
    const rpcConfig = { url, username, password }
    const walletRPC = new WalletRPC(rpcConfig)
    ;(async () => {
      const pubKey = await walletRPC.getViewKeyAsString(address)
      const privKey = await walletRPC.getPrivKeyAsString(address)
      setPublicKey(pubKey)
      setPrivateKey(privKey)
    })()
  }, [address])```

Hi @ngvtuan, could you please take a look this code?
if it not disable it, then it will cause the error by `prettier`